how-to-docs/drone-how-to.md

```diff
@@ -327,6 +327,9 @@ Drone will only execute that step when a new pull request is raised (and when pu
 
 ### Deploying to ACP
 
+> Please note that this section assumes that you already have kube files to work with (specifically, deployment, service and ingress files).
+> Examples of these files can be found in the [platform introduction guide](https://github.com/UKHomeOffice/application-container-platform/blob/master/developer-docs/platform_introduction.md#deploy-to-kubernetes).
```

Comment:
Change to using this project as a template: https://github.com/UKHomeOffice/kube-signed-commit-check